docs/Types.md

```diff
@@ -17,21 +17,21 @@ Every FRAME chain exposes metadata about its types and capabilities. This metada
 As always, our first step is to bring Capi into scope.
 
 ```ts
-import * as C from "https://deno.land/x/capi/mod.ts"
+import * as C from "https://deno.land/x/capi/mod.ts";
 ```
 
-Now let's fetch the metadata.
+As an example, let's connect to Polkadot and fetch the chain's metadata for the present highest block.
```

Comment:
Is this true? Of is this somehow statically defined in capi?

Reply:
I'm not sure exactly what you mean. Can you please re-word this question?

Reply:
Am I correct in assuming that the example here using Polkadot metadat is gathered at ruhtime from the network? So this actually is an RPC call that connects to some default node?

Reply:
> Am I correct in assuming that the example here using Polkadot metadata is gathered at runtime from the network? So this actually is an RPC call that connects to some default node?

Correct